src/response.rs

```diff
@@ -44,7 +44,33 @@ impl fmt::Debug for Response {
 }
 
 impl Response {
-    /// Get the final `Url` of this response.
+    /// Get the final `Url` of this `Response`.
+    ///
+    /// ```rust
+    /// # extern crate reqwest;
+    /// # #[macro_use] extern crate error_chain;
+    /// # use reqwest::{Client, Url};
+    /// #
+    /// # error_chain! {
+    /// #     foreign_links {
+    /// #         Reqwest(reqwest::Error);
+    /// #         Url(reqwest::UrlError);
+    /// #     }
+    /// # }
+    /// #
+    /// # fn run() -> Result<()> {
+    /// let url = Url::parse("http://httpbin.org")?;
+    /// let client = Client::new()?;
+    /// let resp = client.get(url.clone())?.send()?;
+    /// assert_eq!(resp.url(), &url);
+    /// # Ok(())
+    /// # }
+    /// # fn main() {
```

Comment:
This will end up making the doctests make an actual network request, which is a bummer. However! This can be changed to a smaller amount of hidden code, and still show what we want.

```
/// # fn run() -> Result<(), Box<::std::error::Error>> {
/// let res = reqwest::get("http://httpbin.org/")?;
/// assert_eq!(res.url().as_str(), "http://httpbin.org/");
/// # }
```

That's all! Also, thinking about the use of checking the url of the Response, this example also could be useful to show how you can see what the final destination url was:

```
/// # fn run() -> Result<(), Box<::std::error::Error>> {
/// let res = reqwest::get("http://httpbin.org/redirect/1")?;
/// assert_eq!(res.url().as_str(), "http://httpbin.org/get");
/// # }
```